### PR TITLE
Revert "Create att"

### PR DESCRIPTION
**Pull request by XQLYM5**
Reverts yellowrook1/-img-0#11a

### CODE REVIEW

File: att [hunk 1]
```diff
@@ -1 +0,0 @@
-a
```
Comment (XQLYM) [inline att:1]: aa.

Comment (XQLYM) [reply att:1]: a.

**Review by XQLYM5 — COMMENTED** [review]
as

**Review by yellowrook1 — CHANGES_REQUESTED** [review]
k

**Review by yellowrook1 — COMMENTED** [review]
as

**Review by yellowrook1 — APPROVED** [review]

**Pull request closed without merging**